src/main/java/com/backbase/ct/bbfuel/client/limit/LimitsPresentationRestClient.java

```diff
@@ -26,17 +27,17 @@
     public void init() {
         setBaseUri(config.getPlatform().getGateway());
         setVersion(SERVICE_VERSION);
-        setInitialPath(LIMITS_PRESENTATION_SERVICE);
+        setInitialPath(LIMITS_PRESENTATION_SERVICE + "/" + CLIENT_API);
     }
 
-    public Response createPeriodicLimit(PeriodicLimitsPostRequestBody periodicLimitsPostRequestBody) {
+    public Response createPeriodicLimit(CreateLimitRequestBody periodicLimitsPostRequestBody) {
         return requestSpec()
             .contentType(ContentType.JSON)
             .body(periodicLimitsPostRequestBody)
             .post(getPath(ENDPOINT_PERIODIC));
     }
 
-    public Response createTransactionalLimit(TransactionalLimitsPostRequestBody transactionalLimitsPostRequestBody) {
+    public Response createTransactionalLimit(CreateLimitRequestBody transactionalLimitsPostRequestBody) {
```

Comment:
this endpoint has been deprecated and will be removed in 2.18.0 so better use the new one now

Reply:
we agreed to postpone for now (will do later)